src/NUnitEngine/nunit.engine.tests/Internal/AddinsFileReaderTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using NUnit.Engine.Internal.FileSystemAccess;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace NUnit.Engine.Internal.Tests
+{
+    /// <summary>
+    /// Tests the implementation of <see cref="AddinsFileReader"/>.
+    /// </summary>
+    [TestFixture]
+    public class AddinsFileReaderTests
+    {
+        private readonly string[] content1 = new string[]
+        {
+            "# This line is a comment and is ignored. The next (blank) line is ignored as well.",
+            "",
+            "*.dll                   # include all dlls in the same directory",
+            "addins/*.dll            # include all dlls in the addins directory too",
+            "special/myassembly.dll  # include a specific dll in a special directory",
+            "some/other/directory/  # process another directory, which may contain its own addins file",
+            "# note that an absolute path is allowed, but is probably not a good idea in most cases",
+            "c:\\windows\\absolute\\directory",
+            "/unix/absolute/directory",
+            "\\transform\\backslash\\to\\slash"
+        };
+
+        [Test]
+        public void Inheritance()
+        {
+            Assert.True(typeof(IAddinsFileReader).IsAssignableFrom(typeof(AddinsFileReader)));
```

Comment:
```suggestion
            Assert.That(typeof(IAddinsFileReader), Is.AssignableFrom(typeof(AddinsFileReader)));
```

How about this format, for a more specific error message? 🙂

Reply:
I tried it but somehow when using `Assert.That` with `Is.AssignableFrom` the test fails. For now, I extended the `Assert.True` with a message to improve readability in case of an error.

But I will try to figure out why
`Assert.True(typeof(IAddinsFileReader).IsAssignableFrom(typeof(AddinsFileReader)));` passes and `Assert.That(typeof(IAddinsFileReader), Is.AssignableFrom(typeof(AddinsFileReader)));` fails.

Reply:
Thank you @CharliePoole for explaining `Is.AssignableFrom` to me. But as Charlie said, since the point of an inheritance test is questionable, I removed it.